src/Utils/matrix_exponential.jl

```diff
@@ -0,0 +1,61 @@
+# currently supported packages
+const SUPPORTED_EXPONENTIAL_PACKAGES = [:ExponentialUtilities, :Expokit]
+
+# =============
+# Global option
+# =============
+
+global exponential_backend = missing  # global state of the exponential backend
+
+function set_exponential_backend!(backend::Module)
+    global exponential_backend = Val(Symbol(backend))
+end
+
+function get_exponential_backend()
+    ismissing(exponential_backend) && error("the exponential backend is not loaded")
+    return exponential_backend
+end
+
+function _expmat(t, A, b)
```

Comment:
packages call this `expmv` (or `expv`) where the `v` stands for the vector (in our notation, it is `b`)

then i would suggest to name this `_expv` or `_expmv`